packages/forms/src/FormSkeleton.js

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
+
 import PropTypes from 'prop-types';
-import Skeleton from '@talend/react-components/lib/Skeleton';
+
+import { SkeletonInput, SkeletonButton } from '@talend/design-system';
+
 import theme from './FormSkeleton.scss';
 
 export default function FormSkeleton({ displayMode, actions }) {
 	// null/undefined actions prop will display default buttons
 	const hasButtons = displayMode !== 'text' && actions?.length !== 0;
 	return (
-		<div className={`${theme.container} tc-skeleton-heartbeat`} aria-busy>
+		<div className={theme.container} aria-busy>
 			<div className={theme['form-content']}>
 				<div className={theme['form-content-wrapper']}>
```

Comment:
Why not a Stack?

Reply:
because the whole css relies on classNames/overrides in drawer mode, and I can't put my finger in it ATM.. baby step